genie/process_mutation.py

```diff
@@ -226,18 +227,110 @@ def process_mutation_workflow(
         syn=syn,
         center=center,
         maf_tableid=maf_tableid,
-        annotated_maf_path=annotated_maf_path,
+        annotated_maf_path=annotation_paths.merged_maf_path,
         flatfiles_synid=flatfiles_synid,
         workdir=workdir,
     )
 
-    return annotated_maf_path
+    full_error_report = concat_annotation_error_reports(
+        center=center,
+        input_dir=annotation_paths.error_dir,
+    )
+
+    store_annotation_error_reports(
+        full_error_report=full_error_report,
+        syn=syn,
+        errors_folder_synid=genie_config["center_config"][center]["errorsSynId"],
+        release="test",
+    )
+    return annotation_paths.merged_maf_path
+
+
+def create_annotation_paths(center: str, workdir: str) -> namedtuple:
```

Comment:
Nice use of `namedtuple` here - I personally think this is a great example of when to use something like this over a `class` to define the structure of `Filepaths`.  That said - nothing wrong with classes either.

What was your rationale in picking `namedtuple` or `class`?

Reply:
I picked `namedtuple` mainly because of its immutability and simplistic, readable structure since we have the fixed set of filepaths we plan to use and we don't plan to change (or need any additional functionality as far as i can forsee) those filepaths in the course of the code. And we don't need to access those filepaths except for this portion of the code.

Reply:
Agreed!